docs/flow diagrams/create-resource.md

```diff
@@ -0,0 +1,34 @@
+```mermaid
+sequenceDiagram
+    participant Client
+    participant FhirServer
```

Comment:
I haven't used mermaid before. So just curious why this file has `participant` values whereas the others don't have it. The VS Code previews did not seem to show any difference.

Reply:
It isn't required. I haven't used it before either, when I started the first example in the documentation had them so I put them in, then later I saw an example without it. Forgot to clean up the first one, thanks for catching it!